### PR TITLE
egrep is deprecated

### PR DESCRIPTION
**Pull request by simondeziel**
egrep has been deprecated since 2007 and warns it's obsolete since:
https://git.savannah.gnu.org/cgit/grep.git/commit/?id=a9515624709865d480e3142fd959bccd1c9372d1

### CODE REVIEW

File: dehydrated [hunk 1]
```diff
@@ -126,7 +126,7 @@ jsonsh() {
     fi
 
     # shellcheck disable=SC2196
```
Comment (txt-file) [inline dehydrated:128]: The shellcheck comment/command should be removed as it is not needed anymore.

**Review by txt-file — CHANGES_REQUESTED** [review]

File: dehydrated (hunk 1 above)
Comment (simondeziel) [reply dehydrated]: Thanks, done.

Comment (txt-file) [reply dehydrated]: That was fast. :smile:

**Review by danimo — APPROVED** [review]

**Comment by lukas2511** [comment]
merged to master branch, thx

**Pull request closed without merging**